README.md

```diff
@@ -185,12 +185,12 @@ For organizational purposes, register endpoints ahead of time:
 
 ```javascript
 
-extensionInput.register('sidebar-left');
-extensionInput.register('main');
-extensionInput.register('footer');
-extensionInput.register('foo');
-extensionInput.register('bar');
-extensionInput.register('shizzle');
+extensionRegistry.register('sidebar-left');
```

Comment:
if we make it extensionRegistry.add it should probably be
extensionRegistry.remove

On Wed, Feb 10, 2016 at 3:57 PM, Ben Petersen <notifications@github.com>
wrote:

> In README.md
> <https://github.com/openshift/angular-extension-registry/pull/32#discussion_r52523652>
> :
>
> > @@ -185,12 +185,12 @@ For organizational purposes, register endpoints ahead of time:
> >
> >  ```javascript
> >
> > -extensionInput.register('sidebar-left');
> > -extensionInput.register('main');
> > -extensionInput.register('footer');
> > -extensionInput.register('foo');
> > -extensionInput.register('bar');
> > -extensionInput.register('shizzle');
> > +extensionRegistry.register('sidebar-left');
>
> Yeah, def feels redundant with "Registry.register" [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/angular-extension-registry/pull/32/files#r52523652>
> .
>